custom_otello_linter/visitors/scenario_and_steps_checkers/screenshot_label_and_func_checker.py

```diff
@@ -0,0 +1,54 @@
+from typing import List
+
+from flake8_plugin_utils import Error
+
+from custom_otello_linter.abstract_checkers import StepsChecker
+from custom_otello_linter.errors import (
+    MissingMakeScreenshotFuncCallError,
+    MissingScreenshotsAllureLabelError
+)
+from custom_otello_linter.helpers.find_make_screenshot_calls import (
+    find_make_screenshot_calls
+)
+from custom_otello_linter.visitors.scenario_visitor import (
+    Context,
+    ScenarioVisitor
+)
+
+
+@ScenarioVisitor.register_steps_checker
+class ScreenshotsLabelAndFuncChecker(StepsChecker):
+
+    def check_steps(self, context: Context, config) -> List[Error]:
+        errors = []
+
+        allure_decorator = self.get_allure_labels_decorator(context.scenario_node)
+        has_screenshot_label = False
+        has_screenshot_func = False
+
+        # Проверяем, есть ли в списке лейблов SCREENSHOTS
+        if allure_decorator:
+            allure_tags = self.get_allure_labels_names(allure_decorator)
+            has_screenshot_label = "SCREENSHOTS" in allure_tags
+
+        # Проверяем наличие вызова функции make_screenshot_for_comparison
+        for step in context.steps:
+            make_screenshot_calls = find_make_screenshot_calls(step.body)
+            if make_screenshot_calls:
```

Comment:
```suggestion
            if get_make_screenshot_calls(step):
```
Переменная make_screenshot_calls все равно дальше не используется, а для одного условия можно и просто вызвать функцию